src/commands/TeamsProductionMessaging.ts

```diff
@@ -0,0 +1,87 @@
+import axios from "axios"
+import yargs from "yargs"
+
+export interface TeamsProductionMessagingOptions {
+  projectName: string
+  platform: string
+  buildUrl: string
+  buildNumber: number
+  webhook: string
+}
+
+export class TeamsProductionMessaging {
+  public async run(options: TeamsProductionMessagingOptions): Promise<void> {
+    if (!this.isUrlValid(options.webhook)) {
+      console.error("The given webhook is not valid.")
+      process.exit(1)
+    }
+
+    if (!this.isUrlValid(options.buildUrl)) {
+      console.error("The given buildUrl is not valid.")
+      process.exit(1)
+    }
+
+    const platformIdentifier = options.platform.toUpperCase()
+    const storeName = platformIdentifier === "IOS" ? "App Store" : "Google Play Store"
+    const messageContents = {
+      title: `${options.projectName}: New release is no available in the ${storeName} [${platformIdentifier}]`,
+      summary: `New Release - ${platformIdentifier}`,
+      text: `New Release ${options.buildNumber} - ${platformIdentifier} <br/> 
+      The newly released version is now available in the ${storeName} or can alternatively be directly downloaded below.<br/> `,
```

Comment:
```suggestion
      The newly released version is now available in the ${storeName}<br/> `,
```

Reply:
it can't ;)

Reply:
Below current state of the messages.
![image](https://github.com/js-soft/codemagic-tools/assets/34318568/fd9306e2-2bc8-4eab-b8ed-2949bce87c31)

Reply:
Production message:
![image](https://github.com/js-soft/codemagic-tools/assets/34318568/01b5483a-0817-422a-8479-87f07da67d0a)